doc/circulation/actions.md

```diff
@@ -20,11 +20,7 @@
 1. :100: __ADD_REQUEST_4__: item in_transit (IN_TRANSIT_FOR_PICKUP)
     1. __ADD_REQUEST_4_1__: request patron = current loan patron → request denied
     1. __ADD_REQUEST_4_2__: request patron != current loan patron →  (add loan PENDING)
-1. :100: __ADD_REQUEST_5__: item in_transit (IN_TRANSIT_TO_HOUSE)
-    1. __ADD_REQUEST_5_1__: PENDING loan does not exist →  (add loan PENDING)
-    1. __ADD_REQUEST_5_2__: PENDING loan exists
-        1. __ADD_REQUEST_5_2_1__: request patron = current loan patron → request denied
-        1. __ADD_REQUEST_5_2_2__: request patron != current loan patron →  (add loan PENDING)
+1. :100: __ADD_REQUEST_5__: item in_transit (IN_TRANSIT_TO_HOUSE) →  (add loan IN_TRANSIT_FOR_PICKUP, item is: in_transit) [cancel previous loan]
```

Comment:
Maybe it is simpler to have here `add loan pending` instead of `cancel loan + add loan IN_TRANSIT_FOR_PICKUP`.
because normally the item is in transit once it arrives at the library , the action `checkin` does the work for you.